tests/acceptance/features/coreApiWebdavProperties1/copyFile.feature

```diff
@@ -282,9 +285,72 @@ Feature: copy file
     And user "Alice" has created folder "FOLDER/sample-folder"
     When user "Alice" copies folder "/FOLDER" to "/Shares/sharedfile1.txt" using the WebDAV API
     Then the HTTP status code should be "204"
+    # Alice now sees the content of "her" folder as folder "/Shares/sharedfile1.txt"
+    # The share that she received from Brian has "automatically" gone into the "declined" state
     And as "Alice" folder "/FOLDER/sample-folder" should exist
     And as "Alice" folder "/Shares/sharedfile1.txt/sample-folder" should exist
     And user "Alice" should not have any received shares
+    And the sharing API should report to user "Alice" that these shares are in the declined state
+      | path                    |
+      | /Shares/sharedfile1.txt |
+    # Brian still has his original sharedfile1.txt and can see that it is shared with Alice
+    And as "Brian" file "sharedfile1.txt" should exist
+    And the content of file "sharedfile1.txt" for user "Brian" should be "file to share"
+    When user "Brian" gets all shares shared by him using the sharing API
+    Then the OCS status code should be "100"
+    And the HTTP status code should be "200"
+    And file "/Shares/sharedfile1.txt" should be included in the response
+    When user "Alice" accepts share "/sharedfile1.txt" offered by user "Brian" using the sharing API
+    Then the OCS status code should be "100"
+    And the HTTP status code should be "200"
+    # now Alice has "her" folder as "/Shares/sharedfile1.txt"
+    # and has the shared file from Brian as "/Shares/sharedfile1 (2).txt"
+    And as "Alice" folder "/Shares/sharedfile1.txt" should exist
+    And as "Alice" folder "/Shares/sharedfile1.txt/sample-folder" should exist
+    And as "Alice" file "/Shares/sharedfile1 (2).txt" should exist
+    And the content of file "/Shares/sharedfile1 (2).txt" for user "Alice" should be "file to share"
+    Examples:
+      | dav_version |
+      | old         |
+      | new         |
+
+
+  Scenario Outline: copy a file over the top of an existing file received as a user share
+    Given using <dav_version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
```

Comment:
we already have this  test:
https://github.com/owncloud/ocis/blob/10bf7f13763bf0faff6bae20fbd74fae547239c7/tests/acceptance/features/coreApiWebdavProperties/copyFile.feature#L216